virttest/utils_windows/drive.py

```diff
@@ -0,0 +1,39 @@
+"""
+Windows drive utilities
+"""
+
+from . import wmic
+
+
+def _logical_disks(session, cond=None, props=None):
+    cmd = wmic.make_query("LogicalDisk", cond, props,
+                          get_swch=wmic.FMT_TYPE_LIST)
+    return wmic.parse_list(session.cmd(cmd, timeout=120))
+
+
+def get_hard_drive_letter(session, label):
```

Comment:
seems dup of function utils_misc.get_win_disk_vol()

Reply:
Yes, I knew that, I planed to replace that with this one.